climsight.py

```diff
@@ -435,6 +666,8 @@ def extract_climate_data(lat, lon, _hist, _future):
             elevation=str(elevation),
             current_land_use=current_land_use,
             soil=soil,
+            biodiv=biodiv,
+            population = population,
```

Comment:
We have to check what is transferred here - looks like it's just part of the table. If your intention to transfer the whole table - we probably should think about it a bit more, I would do some kind of summary.